source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,83 @@
+.. _kotlin-sync-download-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install {+language+}
+
+      Make sure that your system has {+language+} installed and running
+      on JDK 1.8 or later. For more information on getting started with
+      {+language+} and JVM development, see `Get started with Kotlin/JVM
+      <{+kotlin-docs+}/docs/jvm-get-started.html>`__ in the {+language+}
+      language documentation.
+
+   .. step:: Create a {+language+} Project
+
+      This guide shows you how to add the {+driver-short+} dependencies
+      by using Gradle or Maven. We recommend that you use an integrated development
+      environment (IDE) such as IntelliJ IDEA or Eclipse IDE to configure
+      Gradle or Maven to build and run your project.
```

Comment:
The instructions in step 1 dictate using IntelliJ to create a Kotlin project, so a reader who gets to step 2: Create a Project might have already created the project.

Suggestion would be maybe to reconfigure this to be installing the IDE in step 1 and linking to the Kotlin docs to create the project in step 2? Happy to discuss further.

Reply:
How about we get rid of step 1 and move that information into step 2?

Reply:
That also works!